Resources/Prototypes/Catalog/VendingMachines/Inventories/salvage.yml

```diff
@@ -1,14 +1,18 @@
 ﻿- type: vendingMachineInventory
   id: SalvageEquipmentInventory
   startingInventory:
-    Crowbar: 8
+    WeaponCrusherGlaive: 5
+    WeaponProtoKineticAccelerator: 5
+    WeaponCrusher: 5
+    WeaponCrusherDagger: 5
+    MiningDrill: 8
```

Comment:
these should be loot/obtainable imo

Reply:
specifically the drill i think

Reply:
Don't we get those for free on Prospector anyway?

Reply:
Not sure about completely removing them from the pool, as we can get them on ship anyway, and some players (myself included) might simply buy a ship, such as Prospector to grab the drill/glaive, and then sell it again

Because having it is worth it

I've slightly limited their amount instead